index.html

```diff
@@ -0,0 +1,442 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Blinkit</title>
+</head>
+<body>
+    <div class="navbar">
+        <div class="leftNav">
+            <span class="blink">blink</span>
```

Comment:
wrap this blinkit with div because this should be rendered as combined value. If you change the flex direction of  `leftNav` to columns then it will break